kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -362,7 +362,7 @@ window.ToggleNavbarView = BaseView.extend ({
     initialize: function() {
 
         _.bindAll(this);
-        this.listenTo(this.model, "change", this.render);
+        this.listenTo(this.model, "change:client_server_time_diff", this.render);
```

Comment:
The render function is not being called. Perhaps you could even remove this line entirely? Alternatively, which attributes on the model are used when rendering? You could trigger another event when just those attributed are updated and listen for that.